src/django_clickhouse/migration_operators.py

```diff
@@ -0,0 +1,48 @@
+from infi import clickhouse_orm
+from .configuration import config
+
+
+class HintParamMixin:
+    def __init__(self, *args, hints=None, **kwargs):
+        if hints is None:
+            hints = dict()
+        self.hints = hints
```

Comment:
Nit: I'd prefer creating dictionaries with `{}` and shorter syntax for existance check
```suggestion
        self.hints = hints or {}
```